atom/app/atom_main_delegate.cc

```diff
@@ -183,6 +187,9 @@ bool AtomMainDelegate::BasicStartupComplete(int* exit_code) {
   // Disable the ActiveVerifier, which is used by Chrome to track possible
   // bugs, but no use in Electron.
   base::win::DisableHandleVerifier();
+
+  if (IsBrowserProcess(command_line))
+    base::win::PinUser32();
```

Comment:
Did we investigate how chrome doesn't have to deal with calling this api ? They seem to add it only for tests.

Reply:
I'm pretty sure Chrome gets around this by opening a window as soon as it boots, which causes user32.dll to get loaded earlier, whereas we open a window only via JS.

Reply:
Any window chrome opens will be after the `MainMessageLoopRun` and by that time any dll loads will be a blocking call, not sure if thats how they solve it. Either way this change looks good, was just curious about chrome's usage.